### DISCUSSION TITLE
Does the transcription for Japanese language returns Kanji characters?

### DISCUSSION THREAD

**Discussion by nerrewton** [deepgram/community#807] [General help] [answered]
We are using the Japanese (ja) language from the Base Model and users are complaining that the transcription that we get is not accurate. Is the Japanese language able to return Kanji characters?. We just have one variation for Japanese (ja) to use and there is no description on how this specific language (ja) is transcribing the voices.

**Comment by SandraRodgers** [comment, marked as answer]
Hello @nerrewton 

All of our models return Kanji characters (along with hiragana and katakana characters). I've just tested out some audio in our playground and the transcription looks good to me. Do you have specific audio that isn't being transcribed accurately that you would like to share?
